src/install/java/download/gradle.mdx

```diff
@@ -0,0 +1,43 @@
+---
+componentType: default
+headingText: Choose your download option
+---
+
+Here's how to configure Gradle to download and unzip the `newrelic-java.zip` file, which contains all of the New Relic Java agent components:
```

Comment:
Lines 6-13 are a little fuzzy for me + don't make for a smooth reading experience. A suggestion might be: 

Frame the first part of this paragraph as the subject of the unordered list, aka the components in the file that's downloaded. Then instead of leading your procedural steps with a hanging "To do this:", preface your procedural steps with "Here's how to configure....:"

Reply:
Good call!